instrumentation/opentelemetry-annotations-1.0/javaagent/build.gradle.kts

```
@@ -5,6 +5,8 @@ plugins {
 val versions: Map<String, String> by project
 
 dependencies {
+  implementation(project(":instrumentation-annotation-support"))
```

Oh, one more thing that I missed earlier: to ensure that lib ends up in the bootstrap classloader please add it as `implementation` dependency to the `javaagent-bootstrap` module and use `compileOnly` in javaagent instrumentations:

```suggestion
  compileOnly(project(":instrumentation-annotation-support"))
```